Libplanet/Net/Swarm.cs

```diff
@@ -267,6 +267,9 @@ public void Dispose()
         /// the dial-up is cancelled after this timeout, and it tries another peer.
         /// If <c>null</c> is given it never gives up dial-ups.
         /// </param>
+        /// <param name="millisecondsBroadcastBlockInterval">
+        /// The time period of broadcasting chain tip.
+        /// </param>
         /// <param name="millisecondsBroadcastTxInterval">
         /// The time period of exchange of staged transactions.
         /// </param>
```

Comment:
```suggestion
        /// <param name="millisecondsBroadcastBlockInterval">Time interval between each broadcast
        /// of chain tip.</param>
        /// <param name="millisecondsBroadcastTxInterval">Time interval between each broadcast
        /// of staged transactions.</param>
```

Same as the other suggestion.